crates/turborepo-lib/src/boundaries/mod.rs

```diff
@@ -363,7 +381,25 @@ impl Run {
             // Visit the AST and find imports
             let mut finder = ImportFinder::default();
             module.visit_with(&mut finder);
+            let mut warning_count = 0;
             for (import, span, import_type) in finder.imports() {
+                // If the import is prefixed with `@boundaries-ignore`, we ignore it, but print
+                // a warning
+                if Self::is_ignored_import(&comments, *span) {
```

Comment:
Can we avoid having the display logic inside of `check_package` and instead have `check_package` return diagnostics and then have a some sort of "reporter" decide on what to do about each diagnostic? I think we'll want to eventually offer a way to disallow this escape hatch (or on CI display all warnings). Should also make it easier to write unit tests for this codepath.

Reply:
Changed it to store the warnings. We could keep the diagnostics and annotate them as disabled, but I think this works fine for now. If we want to disallow the ignore annotation, we can also just make the `is_ignored_import` check return false